rskj-core/src/main/java/co/rsk/peg/Bridge.java

```diff
@@ -1165,9 +1165,6 @@ public boolean increaseLockingCap(Object[] args) throws BridgeIllegalArgumentExc
         logger.trace("increaseLockingCap");
 
         Coin newLockingCap = BridgeUtils.getCoinFromBigInteger((BigInteger) args[0]);
-        if (newLockingCap.getValue() <= 0) {
```

Comment:
This doesn't affect Bridge tests?

Reply:
Done! Fixed, refactored, and improved the `increaseLockingCap`-related test in the `BridgeTest.java`.